src/java.base/share/classes/java/lang/Module.java

```diff
@@ -203,9 +206,7 @@ public String getName() {
      * class loader. </p>
      *
      * @return The class loader for this module
-     *
-     * @throws SecurityException
-     *         If denied by the security manager
+     * @throws SecurityException If denied by the security manager
```

Comment:
The exception messages are aligned under the exceptions in this class so I assume you didn't mean to change this.